iaso/models/org_unit.py

```diff
@@ -648,10 +640,74 @@ def requested_fields(self) -> typing.List[str]:
         `prefetch_related` of m2m are required when used in bulk.
         """
         requested = []
-        for name in self.new_fields:
+        for name in self.get_new_fields():
             field = getattr(self, name)
             is_m2m = field.__class__.__name__ == "ManyRelatedManager"
             is_requested = field.exists() if is_m2m else field
             if is_requested:
                 requested.append(name)
         return requested
+
+    def reject(self, user: User, rejection_comment: str) -> None:
+        self.reviewed_at = timezone.now()
+        self.reviewed_by = user
+        self.status = self.Statuses.REJECTED
+        self.rejection_comment = rejection_comment
+        self.save()
+
+    def approve(self, user: User, approved_fields: typing.List[str]) -> None:
+        self.__apply_changes(user, approved_fields)
+        self.reviewed_at = timezone.now()
```

Comment:
We keep all dates in UTC in the backend (or at least, that's the intent). Does this do that?

Reply:
Yes, UTC is always used in the code. It's converted to local time only when interacting with end users.